db/migrate/20140109053431_job_text_upgrade.rb

```diff
@@ -1,5 +1,5 @@
 class JobTextUpgrade < ActiveRecord::Migration
   def change
-    change_column :jobs, :output, :text, :limit => 1.gigabyte - 1
+    change_column :jobs, :output, :text, :limit => 1.gigabyte / 4 - 1
```

Comment:
1Gb - 1byte is the max that postgresql allows.I'm a bit wary of reducing the limit in case there is old data (either ours or someone else's) that is too large